install_shcmember.yml

```diff
@@ -6,9 +6,9 @@
     - conf/web
     - conf/server
     - group/shcmember
+    - conf/authentication
```

Comment:
we should consider including the conf/authentication role in configure_shcmember.yml instead